### PR TITLE
llms: convert() add 'revision' argument

### PR DESCRIPTION
**Pull request by Fihade**

### CODE REVIEW

File: llms/mlx_lm/utils.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def _get_classes(config: dict):
     return arch.Model, arch.ModelArgs
 
 
-def get_model_path(path_or_hf_repo: str) -> Path:
+def get_model_path(path_or_hf_repo: str, revision: Optional[str] = None) -> Path:
```
Comment (awni) [inline llms/mlx_lm/utils.py:60]: Thanks, could you add it to the docstring as well?

**Comment by awni** [comment]
@Fihade I'm waiting on you here, let me know when it's ready for review again.

**Comment by Fihade** [comment]
> i

**Comment by Fihade** [comment]
> @Fihade I'm waiting on you here, let me know when it's ready for review again.

Okay

**Comment by Fihade** [comment]
I've updated

File: llms/README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ from mlx_lm import convert
 
 upload_repo = "mistralai/Mistral-7B-Instruct-v0.1"
 
-convert("mistralai/Mistral-7B-v0.1", quantize=True, upload_repo=upload_repo)
+convert("mistralai/Mistral-7B-v0.1", quantize=True, upload_repo=upload_repo, revision="main")
```
Comment (awni) [inline llms/README.md:51]: Could you remove the `revision` parameter here? I like to keep the examples simple with the defaults implicit.

Comment (Fihade) [reply llms/README.md:51]: ok

**Review by awni — APPROVED** [review]
Thanks!

**Pull request merged**